### DISCUSSION TITLE
Why does it say enabled globally?

### DISCUSSION THREAD

**Discussion by YusufDahir** [glenn2223/vscode-live-sass-compiler#247] [Q&A] [answered]
i tried to use this software but it is showing me, this extension is enabled globally. What does that mean?

**Comment by glenn2223** [comment, marked as answer]
When you install an extension, not related to a workspace, by default, it will be enabled globally.

It basically means that it will try to activate my extension on every project you open EVER! 😂

You can opt to disable an extension globally. Then turn it back on later, either globally or specifc to a workspace. You can read more about [disabling an extension](https://code.visualstudio.com/docs/editor/extension-marketplace#_disable-an-extension) from VS Code's docs
